test/options.test.js

```diff
@@ -44,9 +44,9 @@ describe('options', () => {
 
     afterAll((done) => {
       if (server) {
-        server.close(() => {
-          done();
-        });
+        server.close(done);
+      } else {
+        done();
```

Comment:
`done` was not called if the `server` was `null`, causing `Exceeded timeout of 20000 ms for a hook.`.